yoti-sdk-api/src/main/java/com/yoti/api/attributes/AttributeConstants.java

```diff
@@ -0,0 +1,35 @@
+package com.yoti.api.attributes;
+
+public final class AttributeConstants {
+
+    private AttributeConstants() {
+
+    }
+
+    public static final class HumanProfileAttributes {
+
+        public static final String FAMILY_NAME = "family_name";
+        public static final String GIVEN_NAMES = "given_names";
+        public static final String FULL_NAME = "full_name";
+        public static final String DATE_OF_BIRTH = "date_of_birth";
+        public static final String AGE_OVER = "age_over:";
+        public static final String AGE_UNDER = "age_under:";
+        public static final String GENDER = "gender";
+        public static final String POSTAL_ADDRESS = "postal_address";
+        public static final String STRUCTURED_POSTAL_ADDRESS = "structured_postal_address";
```

Comment:
is structured_postal_address supposed to contain all the remaining address attributes?

building_number	
building_name
sub_building_name
address_line_1
address_line_2
address_line_3
town
country
post_code

Reply:
so, the answer is yes :)